src/Native/build-native.sh

```diff
@@ -364,6 +364,11 @@ if [[ $__ClangMajorVersion == 0 && $__ClangMinorVersion == 0 ]]; then
             __ClangMajorVersion=3
             __ClangMinorVersion=6
         fi
+    elif [[ "$__HostOS" == "Linux" && \
```

Comment:
@janvorli - can you check this out?

Honestly, I'm not sure why we are special casing Ubuntu 16.04 here, but not other versions of Ubuntu, or other distros.

Reply:
That is a good question I just translated the patch. It was done as part of https://github.com/dotnet/source-build/pull/97. @sergiusens can you help us better understand this change?

Reply:
We are now migrating to building on all platforms with clang 3.9 as the default, basically just waiting for the lab preparations for that being completed. @mikem8361 has pending PRs in corefx and coreclr for that.

Reply:
> We are now migrating to building on all platforms with clang 3.9 

Is that for `master` or `release/2.0.0` or both?  Note this change is for `release/2.0.0` with the plan of merging it back to `master`.

Reply:
There was no discussion about porting it to 2.0.0, but I think it will be needed since that's part of the effort to get dumplings that we can easily debug. @danmosemsft do you agree that we will want to enable building with clang 3.9 in 2.0.0 too?

Reply:
Sorry for the late reply, it is related to PGO, full details here https://github.com/dotnet/source-build/issues/95
I special cased Ubuntu to not be invasive and didn't have any other platform to test on.

Reply:
@sergiusens these changes are causing errors when executing:
```
/mnt/resource/j/workspace/dotnet_source-build/dev_release_2.0/RHEL7.2_Release_prtest/src/coreclr/build.sh: line 879: lsb_release: command not found
```
They are getting ignore in some cases but I expect they are going to fail other things in some case. Where should the lsb_release tool be found?

Reply:
@janvorli I expect shiproom would be nervous about changing what we build with I the servicing branch, at least without evidence the dumpfile problem is quite serious. Right now we do not get dumps from customers generally so my focus had been on dumps from test runs in master.